src/main/java/Anke.java

```diff
@@ -0,0 +1,316 @@
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Anke {
```

Comment:
This is a very long piece of code. We will be looking at class design some time down the line which may mitigate this situation.